### DISCUSSION TITLE
How to send Robot to special position?

### DISCUSSION THREAD

**Discussion by hallerdavid** [copystring/ioBroker.roborock#418] [Q&A]
I want to send Roborock S5 Max and Roborock S8 Pro Ultra to the bin in the kitchen. Can you tell me what is the easiest way to find the correct coordinates with iOS 17 and Roborock App? And how can i send the robot to this coordinates?

Thank you!

**Comment by copystring** [comment]
In the web interface, open the console and click anywhere where you want to send the robot. It'll show you the correct coordinates.

**Comment by hallerdavid** [comment]
got it, thank you! i have now the coordinates. Can you tell me how to send the robot to the coordinates? Just give the coordinates to the correct oject? 

so click the object and enter "[31825,33175]" as value?

**Comment by copystring** [comment]
Yes, put `[31825,33175]` in to the go to command.

**Comment by seh-len** [comment]
Does this still work? Unfortunately, I'm not getting any output at all in the Roborock Map browser console. Not when I just click on the map with the mouse, nor when I use “GoTo Point.”

ps.
I found the entry in the ioBroker log.
